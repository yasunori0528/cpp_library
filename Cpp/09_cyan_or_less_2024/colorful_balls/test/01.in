6
1 1 10000
1 1 14142
3 2
1 1 17320
2 1
3 1
