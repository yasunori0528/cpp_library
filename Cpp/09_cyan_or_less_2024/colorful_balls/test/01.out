10000
14142
